pom.xml

```diff
@@ -72,5 +73,10 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>${commons-cli.version}</version>
+        </dependency>
```

Comment:
Can you move this dependency before the section marked as `<!-- Testing -->` please'

Reply:
Yes!